designs/virtualization/4.6/Upload-image-prior-to-creating-VM/Upload Image prior to creating vm.md

```diff
@@ -0,0 +1,52 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Upload Image (data source) prior to creating a VM
+
+This PR deals with when a user wants to upload an image (data source) prior to creating a virtual machine.
+It needs to be accomplished via the storage/PVC area.
+It should ensure the user understands what the backing PVC does but does not remove the Disk terminology.
+
+The ‘Create PVC’ button will change into a dropdown, so users will be able to select between 'New Persistent Volume Claim' or ‘New Persistent Volume Claim with data upload’ from this dropdown.
+By creating a dropdown and adding this new action, we'll highlight this action as a new feature (in an existing form) so the users will be able to easily find it.
+
+![PVC list with drop down button](img/PVC-dropdown.png)
```

Comment:
@yfrimanm I filed this bug and adjusted the text a bit after talking with @itsptk 
do you mind adjusting the design to align with that? https://bugzilla.redhat.com/show_bug.cgi?id=1862508

Reply:
Done @matthewcarleton & thank you for filing the bug.